Tasks/AzureFileCopyV2/Utility.ps1

```diff
@@ -294,6 +305,9 @@ function Upload-FilesToAzureContainer
     finally
     {
         Handle-AzCopyLogs -isLogsPresent $useDefaultArguments -logsFilePath $azCopyLogFilePath -ErrorAction SilentlyContinue
+        if ((Test-Path -Path $responseFile -PathType Leaf) -eq $true) {
+            Remove-Item -Path $responseFile -Force
+        }
```

Comment:
Should we just do:
Remove-Item -Path $responseFile -Force -ErrorAction SilentlyContinue
instead of If block?

Will prevent task failure if for some reason Remove-Item fails.

Reply:
yes. makes more sense.